gradle/projectDependencyGraph.gradle

```diff
@@ -47,13 +47,15 @@ task projectDependencyGraph {
                 .withType(ProjectDependency)
                 .collect { it.dependencyProject }
                 .each { dependency ->
-          projects.add(project)
-          projects.add(dependency)
-          rootProjects.remove(dependency)
-
-          def graphKey = new Tuple2<Project, Project>(project, dependency)
-          def traits = dependencies.computeIfAbsent(graphKey) { new ArrayList<String>() }
+          if (!(config.name.toLowerCase().contains('androidtest') || config.name.toLowerCase().contains('unittest'))) {
+              projects.add(project)
+              projects.add(dependency)
+              rootProjects.remove(dependency)
 
+              def graphKey = new Tuple2<Project, Project>(project, dependency)
+              def traits = dependencies.computeIfAbsent(graphKey) { new ArrayList<String>() }
+          }
+          
           if (config.name.toLowerCase().endsWith('implementation')) {
             traits.add('style=dotted')
           }
```

Comment:
I think the `if` you're introducing also needs to include this other `if`, otherwise `traits` won't exist here since it's defined within the scope of the first `if`.